src/Processors/Formats/Impl/TabSeparatedRowInputFormat.cpp

```diff
@@ -320,7 +320,7 @@ void registerTSVSchemaReader(FormatFactory & factory)
     }
 }
 
-static std::pair<bool, size_t> fileSegmentationEngineTabSeparatedImpl(ReadBuffer & in, DB::Memory<> & memory, size_t min_chunk_size, bool is_raw, size_t min_rows)
+static std::pair<bool, size_t> fileSegmentationEngineTabSeparatedImpl(ReadBuffer & in, DB::Memory<> & memory, size_t min_chunk_size, bool is_raw, size_t min_rows, size_t max_rows)
```

Comment:
Maybe we can delete min_chunk_size at all?

Reply:
It's assigned to the setting [min_chunk_bytes_for_parallel_parsing](https://github.com/ClickHouse/ClickHouse/pull/41852/files#diff-0eaedd23d42061c60a482485b905f5168cb92e20d14fec9e14f9d7299edc7c09R250) and this setting seems to be useful too.